testing/python/layout/test_tilelang_cute.py

```
@@ -390,18 +383,167 @@ def test_tma_box_validity_via_composite():
+    assert tvm.ir.structural_equal(L, cute.make_identity_layout((4, 4)))
+    for k, s in enumerate(L.stride):
+        assert isinstance(s, cute.ScaledBasis) and s.value == 1 and s.mode == (k,)
+
+
+def test_make_layout_nested_column_row_identity():
+    # Column/row-major and identity over a nested shape produce congruent nested
+    # strides (CuTe compact_col_major / compact_row_major / make_identity_layout).
+    assert tvm.ir.structural_equal(cute.make_column_major_layout((2, (2, 2))), cute.make_layout((2, (2, 2)), stride=(1, (2, 4))))
+    assert tvm.ir.structural_equal(cute.make_row_major_layout((2, (2, 2))), cute.make_layout((2, (2, 2)), stride=(4, (2, 1))))
+    # Identity maps each linear coord to its full nested idx2crd coordinate.
+    I = cute.make_identity_layout((2, (2, 2)))
```

Ruff (0.15.20)

[error] Ambiguous variable name: `I`

(E741)